contracts/proposals/IncentiveUpdateExecutor.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.7.5;
+pragma abicoder v2;
+
+import {DataTypes} from '../utils/DataTypes.sol';
+import {IAaveEcosystemReserveController} from '../interfaces/IAaveEcosystemReserveController.sol';
+import {IAaveIncentivesController} from '../interfaces/IAaveIncentivesController.sol';
+import {IATokenDetailed} from '../interfaces/IATokenDetailed.sol';
+import {ILendingPoolConfigurator} from '../interfaces/ILendingPoolConfigurator.sol';
+import {ILendingPoolData} from '../interfaces/ILendingPoolData.sol';
+
+contract IncentiveUpdateExecutor {
+
+  address constant AAVE_TOKEN = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
+  address constant ECO_RESERVE_ADDRESS = 0x1E506cbb6721B83B1549fa1558332381Ffa61A93;
```

Comment:
Minor comment but useful for clarity @ncitron . Better a name like `CONTROLLER_ECO_RESERVE` as the called contract is the controller, not the reserve itself holding the funds, and if not it confuses a bit while on code review